eslint.config.js

```
@@ -0,0 +1,101 @@
+      "tsdoc/syntax": "warn",
+
+      // **JSDoc**
+      "jsdocPlugin/require-jsdoc": "warn",
+      "jsdocPlugin/require-param": "warn",
+      "jsdocPlugin/require-returns": "warn",
+      "jsdocPlugin/check-alignment": "warn",
+      "jsdocPlugin/check-indentation": "warn",
+      "jsdocPlugin/require-description": "warn",
+
+      // **Prettier**
+      "prettierPlugin/prettier": "error",
```

ESLint

[error] Replace `"prettierPlugin/prettier":·"error",` with `'prettierPlugin/prettier':·'error'`

(prettierPlugin/prettier)